VideoLocker/res/values/styles.xml

```diff
@@ -292,6 +292,11 @@
         <item name="android:background">@drawable/white_circle</item>
     </style>
 
+    <style name="discussion_profile_image" parent="profile_image">
+        <item name="civ_border_color">@color/edx_brand_primary_base</item>
+        <item name="civ_border_width">@dimen/discussion_image_border_width</item>
```

Comment:
It seems that the parent `profile_image` style is the style for the user profiles module, but it doesn't have it's own border width defined there. I suppose the best design would be to create a specific `user_profile_image` style that extends from this base one, and move all the module-specific attributes in there.

Also, since we have the diameters defined as a dimension, I suppose it should also be defined as the width and height in these styles.